core/routemgmt/getApi.js

```diff
@@ -137,11 +137,11 @@ function readFilteredApiDocument(cloudantDb, designDocId, designDocViewName, par
   return new Promise(function (resolve, reject) {
     cloudantDb.view(designDocId, designDocViewName, params, function(error, response) {
       if (!error) {
-        console.log('success', response);
+        console.log('DB view success', JSON.stringify(response));
         resolve(response);
       } else {
-        console.error('error', JSON.stringify(error));
-        if (error.headers.statusCode == 404) {  // When NOT FOUND, return empty results
+        console.error('DB view error', JSON.stringify(error));
+        if (error.headers && error.headers.statusCode === 404) {  // When NOT FOUND, return empty results
```

Comment:
based on the nano.js code, it seems either are acceptable.  i found examples using both statusCode values... but i think using the top level is a bit more readable